### PR TITLE
Add ability to specify the RELEASE arg in build_docker_remote.sh, and add a release_prebuilt_docker_image.sh script

### PR DESCRIPTION
**Pull request by droazen**

### CODE REVIEW

**Comment by droazen** [comment]
I used the updated scripts in this PR to successfully push the 4.4.0.0 release image, so these are confirmed to be working correctly.

File: build_docker_remote.sh [hunk 1]
```diff
@@ -81,9 +82,22 @@ if [ -z "$DOCKER_IMAGE_TAG" ]; then
...
+  echo "- name: 'gcr.io/cloud-builders/docker'" >> cloudbuild.yaml
+  echo "  args: [ 'build', '-t', '${DOCKER_IMAGE_TAG}', '--build-arg', 'RELEASE=true', '.' ]" >> cloudbuild.yaml
+  echo "- name: 'gcr.io/cloud-builders/docker'" >> cloudbuild.yaml
+  echo "  args: [ 'push', '${DOCKER_IMAGE_TAG}' ]" >> cloudbuild.yaml
+fi
+
 echo "Building image with the tag ${DOCKER_IMAGE_TAG}..."
 
-SUBMIT_COMMAND="gcloud builds submit --tag ${DOCKER_IMAGE_TAG} --timeout=24h --machine-type n1_highcpu_8"
+if [ -n "$RELEASE" ]; then
```
[the remark this answers is not in the document]
Comment (droazen) [reply build_docker_remote.sh]: Fixed. Also added an explicit `--squash` on release

**Comment by droazen** [comment]
Had to remove the `--squash` argument, as I couldn't get it to work with either `gcr.io/cloud-builders/docker` or `docker.io/library/docker`. We'll have to address the issue of image size in separate PRs.

**Review by jamesemery — APPROVED** [review]
This looks better provided that you've tested and the default tagging is working for your use case. This is a more... confusing... way of propagating the tag but at least its consistent.

**Pull request merged**